aten/src/ATen/native/sparse/cuda/SparseCUDAApplyUtils.cuh

```diff
@@ -207,6 +222,45 @@ __global__ void valueSparseIntersectionKernel(
   }
 }
 
+// TODO find a way to parallelize this...
+template <typename Op, typename IndexType, typename Real>
```

Comment:
wait so this kernel is run on entire tensors, i.e., not parallelized?

Reply:
No, not yet. Now the cuda kernel needs to iterate through all indices, one at a time. Do you have any thoughts? Will cuSPARSE be helpful (at least in sparse matrix)?